app/src/main/java/com/celzero/bravedns/service/BraveVPNService.kt

```diff
@@ -598,36 +645,38 @@ class BraveVPNService : VpnService(), ConnectionCapabilityMonitor.NetworkListene
         appInfoRepository.getUIDForUnivWhiteList().observeForever { appInfoList ->
             appWhiteList = appInfoList.associateBy({ it }, { true }).toMutableMap()
         }
-        if(persistentState.orbotEnabledMode != Constants.ORBAT_MODE_NONE){
+        if (persistentState.orbotEnabledMode != Constants.ORBAT_MODE_NONE) {
             get<OrbotHelper>().startOrbot(this)
         }
         blockUDPTraffic = persistentState.udpBlockedSettings
-        privateDNSOverride = persistentState.allowPrivateDNS
-        isScreenLocked = persistentState.getScreenLockData()
+        isScreenLocked = persistentState.isScreenOff
```

Comment:
If the isScreenOff is not persisted then the value is getting lost when the VPN is restarted. 
We were having it in-memory earlier and then changed to shared pref because of the issue.

Reply:
Created issue to remove isScreenOff from PersistentState. https://github.com/celzero/rethink-app/issues/304